client/src/List.svelte

```diff
@@ -153,13 +151,18 @@
 					}
 				}
 				if ("answered" in adj) {
-					if (q.answered === adj.answered) {
+					// Ohhh, how I wish Javascript had a match statement like rust
+					const patch = adj.answered;
+					if (patch.action === "unset" && "answered" in q) {
+						console.info("remove answered property");
+						delete qs[i].answered;
+					} else if (patch.action === "set" && !("answered" in q)) {
```

Comment:
This isn't 100% correct, though probably good enough in practice. In theory at least, I could mark as answered, wait for a sync, unmark, then mark again, and local adjustments wouldn't end up being applied to set the new timestamp. I don't think that matters though.

Reply:
I initially started to write the code to handle those cases, but it complicated the branches quite a bit for a situation that is unlikely to happen (and if it does happen, it should resolve itself on the following sync since we'll use the timestamp from RawQuestions once localAdjustments is marked as resolved). 

I'll let you decide if want to commit as is. If you would prefer a more robust solution let me know and I can switch it up.

Reply:
I think as-is is fine!